string ugv_id
string[] task_ids
float64[] bids
string[] winners  # corresponding ugv_ids
